Dfe.Academies.Academisation.WebApi/Controllers/TransferProjectController.cs

```diff
@@ -52,7 +55,7 @@ public class TransferProjectController : ControllerBase
 				_ => throw new NotImplementedException()
 			};
 		}
-		[HttpPut(Name = "SetTransferProjectTransferDates")]
+		[HttpPut("set-transfer-dates", Name = "SetTransferProjectTransferDates")]
 		[ProducesResponseType(StatusCodes.Status201Created)]
```

Comment:
This was from my work but 201's don't make sense for these sets, I can do that as part of work I'm doing